src/workos.ts

```diff
@@ -73,7 +76,10 @@ export class WorkOS {
       this.options.https = true;
     }
 
-    this.clientId = this.options.clientId ?? process?.env.WORKOS_CLIENT_ID;
+    this.clientId = this.options.clientId;
+    if (!this.clientId && typeof process !== 'undefined') {
+      this.clientId = process?.env.WORKOS_CLIENT_ID;
+    }
```

Comment:
Looked in to how we could replicate the lack of `process` in a jest test to no avail. May be something we want to dig into further to strengthen the worker test.